### PR TITLE
Toggle bar for trips

### PR DESCRIPTION
**Pull request by J-Massmann**

### CODE REVIEW

File: src/components/TripCard.spec.js [hunk 1]
```diff
@@ -1,16 +1,55 @@
 import TripCard from './TripCard.js';
 import { render, screen } from '@testing-library/react';
+import { userEvent } from '@storybook/testing-library';
```
Comment (Quaquaro) [inline src/components/TripCard.spec.js:3]: I think you are not importing the right thing here. The following should be imported.
`import userEvent from '@testing-library/user-event';`

File: src/components/TripCard.stories.js [hunk 2]
```diff
@@ -5,4 +5,19 @@ export default {
   component: TripCard,
 };
 
-export const Default = () => <TripCard />;
```
Comment (Quaquaro) [inline src/components/TripCard.stories.js:8]: that should also work:

`export const Default = Template.bind({});
Default.args = {
    id: "1",
    place: "Barcelona",
    locations: "['Sagrada Familia', 'Park Güell']",
    isTripFuture: true,
  },
};

export const TripCardPast = Template.bind({});
TripCardPast.args = {
    ...Default.args.,
    isTripFuture: "false",
  },
};`

**Review by Quaquaro — COMMENTED** [review]
Check if the TripCard test is running.
I'm not sure about that.

File: src/components/TripCard.spec.js (hunk 1 above)
Comment (J-Massmann) [reply src/components/TripCard.spec.js:3]: Surprisingly the test fails if I don't import it with brackets 😅

File: src/components/TripCard.stories.js (hunk 2 above)
Comment (J-Massmann) [reply src/components/TripCard.stories.js:8]: Yup still working, thanks for the hint :)

**Pull request merged**